.flake8

```diff
@@ -0,0 +1,12 @@
+[flake8]
```

Comment:
https://github.com/NRLMMD-GEOIPS/geoips/blob/main/.config/flake8

This currently matches what's in the geoips repo .config directory.  This is what tests/utils/check_code.sh uses, and the CI uses check_code!

https://github.com/NRLMMD-GEOIPS/geoips/blob/38e95a622e70e172db79d9193e3fea2587de89c1/.github/workflows/doc-lint-test.yaml#L538

https://github.com/NRLMMD-GEOIPS/geoips/blob/38e95a622e70e172db79d9193e3fea2587de89c1/.github/workflows/doc-lint-test.yaml#L506

Reply:
Perhaps we merge this for now, because at least it would provide VSCode settings that are consistent with GeoIPS standards ?